source/acquaman/BioXAS/BioXASMainXASScanConfiguration.h

```diff
@@ -32,6 +32,10 @@ class BioXASMainXASScanConfiguration : public AMStepScanConfiguration, public Bi
 	virtual QString detailedDescription() const;
 	/// Get a nice looking string that contains all the standard information in an XAS scan.  Used when exporting.
 	virtual QString headerText() const;
+    /// Returns the current total estimated time for a scan to complete.
+    double totalTime() const { return totalTime_; }
```

Comment:
The totalTime() and totalTime(bool) methods are also already defined in BioXASScanConfiguration, I think.